### DISCUSSION TITLE
🧩 FEATURE REQUEST: Show share popup after commenting via the Companion Widget

### DISCUSSION THREAD

**Discussion by Kvaibhav01** [dailydotdev/daily#612] [Feature Requests]
### Description

I would like to see the share popup (shown below) when I finish commenting via the Companion Widget.

<img width="320" alt="Daily.dev share popup" src="https://user-images.githubusercontent.com/11731837/173284583-51b2effc-8fd9-4c2a-bc56-f470367c8bf6.png">

### Solution

After I click the "Comment" button, this popup should appear so that I can easily share my thoughts on socials. Right now I am not able to share my comment if I use this widget.

**Comment by idoshamun** [comment]
Hi @Kvaibhav01,

That's a great suggestion! I'm adding to the backlog of companion v2 :)

**Comment by ZaLiTHkA** [comment]
may I request that this be made an optional setting? it's not something that _everybody_ would want (myself included)..

I don't use Twitter at all, I barely use WhatsApp, and my Facebook profile is only for non-development things. besides that, if I _do_ happen upon an article that I really feel the need to share, I do so directly from the Google Chrome address bar "share" link anyway.

**Comment by Kvaibhav01** [comment]
A/B testing with insiders/beta users might help here!

**Comment by ZaLiTHkA** [comment]
> A/B testing with insiders/beta users might help here!

I got an invitation recently to join beta testing for the Daily.dev extension, though I don't see anything different after doing that.. little busy at the moment, so I'll look into that a little more later this evening.

[edit] so.. Gmail ended up assigning a "spam" label to my beta invite email. 🤦🏼‍♂️

**Comment by idoshamun** [comment]
Thank you @ZaLiTHkA, for sharing your perspective. We'll take that into account.
